### ISSUE TITLE
Umbrel & docker documentation outdated

### ISSUE THREAD

**Issue by jvrakor** [accumulator/charge-lnd#73]
Since the upgrade to v0.2.9, the commands specified in https://github.com/accumulator/charge-lnd/blob/master/INSTALL.md#umbrel are no longer correct. What used to work in v0.2.8 no longer works in v0.2.9:
```
umbrel@umbrel:~ $ docker run --rm --network=umbrel_main_network -e GRPC_LOCATION=<IP> -e LND_DIR=/data/.lnd -e CONFIG_LOCATION=/app/charge.config -v /home/umbrel/umbrel/lnd:/data/.lnd -v /mnt/data/upgrades/charge-lnd:/app accumulator/charge-lnd:v0.2.9
usage: charge-lnd [-h] [--lnddir LNDDIR] [--tlscert TLS_CERT_PATH]
                  [--macaroon MACAROON_PATH] [--grpc GRPC]
                  [--electrum-server ELECTRUM_SERVER] [--dry-run] [--check]
                  [-v] -c CONFIG
charge-lnd: error: the following arguments are required: -c/--config
```

**Comment by jvrakor** [comment]
Fixed in https://github.com/accumulator/charge-lnd/pull/74.

**Comment by accumulator** [comment]
Should be fixed in v0.2.10
